tests/rptest/services/w_workload.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2020 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.services.service import Service
+from ducktape.cluster.remoteaccount import RemoteCommandError
+from threading import Event
+import requests
+from time import sleep
+import sys
+from rptest.util import wait_until
+
+
+class WWorkload(Service):
```

Comment:
This would be a good place for a docstring explaining the purpose of the class -- it's especially useful to say when one might use this service as opposed to any of the other producers.

Reply:
removed that script